tests/cockpit-tests

```diff
@@ -31,17 +31,12 @@ if [ ! -d cockpit ]; then
     git clone https://github.com/cockpit-project/cockpit
 fi
 
+# TODO: This needs to move into tests bot
 if [ ! -f $TMPDIR/vm-prep-was-run ]; then
     sudo /usr/local/bin/vm-prep
     touch $TMPDIR/vm-prep-was-run
 fi
 
-if [ ! -d cockpit/node_modules ]; then
```

Comment:
Not saying this is wrong, just wondering what changed that makes this no longer required.